### PR TITLE
Check if name is valid before creating SketchFile

### PR DESCRIPTION
**Pull request by isabarros**
This PR makes SketchFile constructor raise a custom exception called InvalidName in two scenarios:

- When the name provided does not start with a letter or underscore
- When the name provided contains characters that are not alphanumeric except underscore

### CODE REVIEW

File: pyp5js/fs.py [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+from pyp5js.exceptions import SketchDirAlreadyExistException, InvalidName
 
 
 class SketchFiles():
     TARGET_NAME = 'target'
     STATIC_NAME = 'static'
 
     def __init__(self, sketch_name):
+        does_not_start_with_letter = r'^[^a-zA-Z]'
+        contains_non_alphanumerics = r'[^a-zA-Z0-9]'
```
Comment (turicas) [inline pyp5js/fs.py]: "_" is a valid char and is missing in this regexp.

Comment (isabarros) [reply pyp5js/fs.py]: Going to add that @turicas ! Thanks for the reply

File: pyp5js/tests/test_fs.py [hunk 2]
```diff
@@ -95,3 +96,11 @@ def test_create_dirs(self):
...
         with pytest.raises(SketchDirAlreadyExistException):
             self.files.create_sketch_dir()
+
+    def test_raise_exception_when_name_starts_with_numbers(self):
+        with pytest.raises(InvalidName):
+            SketchFiles('123name')
+
+    def test_raise_exception_when_name_contains_non_alphanumeric_chars(self):
+        with pytest.raises(InvalidName):
+            SketchFiles('name&')
```
Comment (berinhard) [inline pyp5js/tests/test_fs.py]: Tests :heart_eyes:  :heart_eyes:  :heart_eyes:

File: pyp5js/fs.py (hunk 1 above)
Comment (berinhard) [reply pyp5js/fs.py]: +1 for the `_` here

**Comment by isabarros** [comment]
@berinhard Updated!

**Pull request merged**